_format/1.1/index.md

```diff
@@ -190,8 +200,18 @@ Here's how an article (i.e. a resource of type "articles") might appear in a doc
 
 #### <a href="#document-resource-object-identification" id="document-resource-object-identification" class="headerlink"></a> Identification
 
-Every [resource object][resource objects] **MUST** contain an `id` member and a `type` member.
-The values of the `id` and `type` members **MUST** be strings.
+Every [resource object][resource objects] **MUST** contain an `id` member and a
+`type` member. As noted above, the only exception is that the `id` member is not
```

Comment:
The exception is spelled out above as follows:

> A resource object **MUST** contain at least the following top-level members:
>
> * `id`
> * `type`
> 
> Exception: The `id` member is not required when the resource object originates at
> the client and represents a new resource to be created on the server.

Reply:
BTW, I'm open to rewording this in both the 1.0 and 1.1 specs so that the exception clause is integrated in the requirement instead of standing on its own.